buildroot-external/patches/occu/0153-WebUI-DynamicResizing.patch

```diff
@@ -0,0 +1,142 @@
+--- occu/WebUI/www/rega/login.htm.orig
++++ occu/WebUI/www/rega/login.htm
+@@ -40,31 +40,10 @@
+           FormSubmit();
+         }
+       }
+-
+-      recalcDivs = function()
+-      {
+-        var wHeight = 0;
+-        if (window.innerHeight)
+-        {
+-          wHeight = window.innerHeight;
+-        }
+-        else if (document.documentElement.offsetHeight)
+-        {
+-          wHeight = document.documentElement.offsetHeight;
+-        }
+-
+-        var wHeaderHeight = 72;
+-        var wFooterHeight = 32;
+-        var wContentHeight = wHeight - wHeaderHeight - wFooterHeight - 27;
+-
+-        $("navbar").setStyle({height: wHeaderHeight + "px"});
+-        $("content").setStyle({height: wContentHeight + "px"});
+-        $("footer").setStyle({height: wFooterHeight + "px"});
+-      }
+     </script>
+   </head>
+   <body>
+-    <table style="width:100%; height:100%;" cellpadding="0" cellspacing="0" id="maintable">
++    <table cellpadding="0" cellspacing="0" id="maintable">
```

Comment:
Any chance to get rid of this old-style `cellspacing` and `cellpadding` and replacing them by CSS statements?

Reply:
It could be done, but I'm unsure if it should be in scope of this PR. I would prefer to have this fix in another one.

Reply:
Applying subsequent changes to the same line in multiple patches can be quite critical and introduce conflicts in applying patches after another. Thus this would introduce hard dependencies between webui patches and don't make them independent anymore.

Reply:
I just thought about another PR on this patch. There is only so much one can do in a day 😊

Reply:
IMHO it would be better to integrate this change also in this PR since it is also changing the style stuff. Or you integrate this into the generic `0039-WebUI-Fix-Style-Glitches` which tries to improve the general styling stuff on the WebUI.

Reply:
should now also be fixed with f98f231